norfair/drawing.py

```diff
@@ -14,8 +14,10 @@ def draw_points(frame, detections, radius=None, thickness=None, color=None):
         thickness = int(max(frame_scale / 7, 1))
     if color is None:
         color = Color.red
-
+    color_is_rand = color==Color.rand
```

Comment:
This check only happens once, so there is no need to define it in a variable, we can just use it at the same time as we define it. Its shorter and clearer.